v2/pkg/passive/sources.go

```diff
@@ -2,6 +2,7 @@ package passive
 
 import (
 	"fmt"
+	"github.com/projectdiscovery/subfinder/v2/pkg/subscraping/sources/leakix"
```

Comment:
You should probably move this under line 33.

Reply:
This was fixed in a previous commit